src/gluonts/dataset/artificial/generate_synthetic.py

```diff
@@ -56,7 +56,7 @@ def write_csv_row(
         if freq_week_start == "W":
             freq_week_start = f"W-{week_dict[timestamp.weekday()]}"
         timestamp = pd.Timestamp(data["start"], freq=freq_week_start)
-        item_id = int(data["item"])
+        item_id = int(data["item_id"])
```

Comment:
The best thing would be to use [gluonts.dataset.field_names.FieldName](https://github.com/awslabs/gluon-ts/blob/ff22722cb3c19779dfefdd5b6d2c0e38ad93e605/src/gluonts/dataset/field_names.py#L15) (not just here, but all over the module, in case other field names are hardcoded)

Reply:
Yes, there are other hardcodings of `item_id` in `recipe.py` and `_base.py` that we should update as well.